### PR TITLE
fixes uplinks not showing role/race restricted items

### LINKED ISSUE #47452: Discount traitor and nukie items
There are no discount items in the nukie and traitor uplink. Not sure if I was extremely unlucky, please try to reproduce it by trying to open the discount item tab. 
Happened several times over the past few days
edit: I left out a word god damit

### PR DESCRIPTION
**Pull request by actioninja**
## About The Pull Request

Not only was this accidentally inverted, this isn't actually a boolean.

Fixes #47452

## Changelog
:cl:
fix: role and race restricted items should show up in the uplink again.
/:cl:

### CODE REVIEW

**Comment by actioninja** [comment]
hold on stray file snuck in somehow

File: tgui-next/packages/tgui/styles/components.scss [hunk 1]
```diff
@@ -1,13 +0,0 @@
...
-@import '../components/LabeledList.scss';
-@import '../components/Layout.scss';
-@import '../components/NoticeBox.scss';
-@import '../components/NumberInput.scss';
-@import '../components/ProgressBar.scss';
-@import '../components/Section.scss';
-@import '../components/Table.scss';
-@import '../components/Tabs.scss';
-@import '../components/TitleBar.scss';
-@import '../components/Tooltip.scss';
```
Comment (stylemistake) [inline tgui-next/packages/tgui/styles/components.scss]: What on earth is happening

Comment (actioninja) [reply tgui-next/packages/tgui/styles/components.scss]: this file in particular sucked

**Review by stylemistake — APPROVED** [review]

**Comment by Tharcoonvagh** [comment]
And discount items?

**Comment by Tharcoonvagh** [comment]
Also role restricted items are fine

**Comment by Cyprex** [comment]
Checked it out locally, this fixes discounts! Needs "Fixes #47452"

**Review by ninjanomnom — APPROVED** [review]

**Pull request merged**